metadata-ws/src/test/resources/features/misc.feature

```diff
@@ -110,3 +110,9 @@ Feature: Miscellaneous functions
       | FILE               | file              | {"name": "nothing important"}                   |
       | SAMPLE             | sample            | {"name": "nothing important"}                   |
       | WEB_RESOURCE       | webResource       | {"resourceUrl": "http://nothing.important.com"} |
+
+  Scenario: access security disabled links
+    When I request non-secure GET /
+    Then the response code should be 200
+    When I request non-secure GET /swagger-ui.html
+    Then the response code should be 200
```

Comment:
Could you please add tests which check for the authorisation features, e. g. creating/patching objects?

Reply:
I have converted all existing test cases with authorisation feature. Please let me know if other scenarios apart from this can be added

Reply:
can you please add negative test cases as well.

Reply:
I am happy with the current test cases. @selvaebi if you agree as well, then please mark this as resolved